tests/common/integration_test.go

```diff
@@ -23,4 +23,10 @@ import (
 
 func init() {
 	testRunner = framework.IntegrationTestRunner
+	clusterTestCases = []testCase{
+		{
+			name:   "NoTLS",
+			config: config.ClusterConfig{ClusterSize: 1},
+		},
+	}
```

Comment:
Does it mean you remove some cases for integration test?

Reply:
Yes, maybe it was worth to mention it more.
With e2e and integration having 1to1 coverage of same tests, we can speed up tests while maintaining same coverage. 
I think this needs a broader discussion, so I will move it to separate PR.